src/components/modal/index.js

```diff
@@ -86,11 +90,26 @@ export default class Modal extends Component {
                 </button>
               </div>
             )}
-            {error && (
+            {formattedError && (
               <div className="flashMessage error">
-                <span>{t(formatError(error))}</span>
+                <span>{t(formattedError)}</span>
               </div>
             )}
+            {isLocal &&
+              formattedError &&
+              formattedError.includes("Failed to load settings from") && (
+                <div>
+                  <button
+                    onclick={e => {
+                      clearSiteURL(e);
+                      clearStoreError(null);
```

Comment:
nitpick: does the need to pass `null`? it doesn't look like there's an accepted argument for this function

Reply:
Whoops! Fixed. 👍